client-v2/src/components/FrontsCAPIInterface/FeedItem.tsx

```diff
@@ -173,22 +187,29 @@ class FeedItem extends React.Component<FeedItemProps> {
                 <Tone> / {startCase(article.frontsMeta.tone)}</Tone>
               )}
             </TagInfo>
-            {article.fields.scheduledPublicationDate && (
-              <ScheduledPublication>
-                {distanceInWordsStrict(
-                  new Date(article.fields.scheduledPublicationDate),
-                  Date.now()
-                )}
-              </ScheduledPublication>
-            )}
-            {article.webPublicationDate && (
-              <FirstPublished>
-                {distanceInWordsStrict(
-                  Date.now(),
-                  new Date(article.webPublicationDate)
-                )}
-              </FirstPublished>
-            )}
+            <RefreshPeriodically rateMs={collectionArticlesPollInterval}>
```

Comment:
Because we can't rely on this feed updating itself as before, the component keeps these times fresh by itself.